kernel/src/engine/arrow_expression.rs

```diff
@@ -867,4 +948,205 @@ mod tests {
         let expected = Arc::new(BooleanArray::from(vec![true, false]));
         assert_eq!(results.as_ref(), expected.as_ref());
     }
+
+    #[test]
+    fn test_create_one() {
+        let expr = Expression::struct_from([
+            Expression::literal(1),
+            Expression::literal(2),
+            Expression::literal(3),
+        ]);
+        let schema = Arc::new(crate::schema::StructType::new([
+            StructField::new("a", DeltaDataTypes::INTEGER, true),
```

Comment:
with a rebase, I think these can be moved over to `StructField::nullable`/`StructField::not_null`.